lib/dirac_mobius.cpp

```diff
@@ -9,6 +9,13 @@ namespace quda {
     memcpy(b_5, param.b_5, sizeof(Complex) * param.Ls);
     memcpy(c_5, param.c_5, sizeof(Complex) * param.Ls);
 
+    double b = b_5[0].real();
+    double c = c_5[0].real();
+    mobius_kappa_b = 0.5 / (b * (m5 + 4.) + 1.);
```

Comment:
Let's add a check for variable coefficients and add an `errorQuda` for the case that there are variable coefficients. I'll accept just comparing the first two coefficients. I assume that would also require a check for kappa vs mass normalization?

Reply:
The check for variable coefficients is done later in this constructor. I don't think the `kappa` and/or mass normalization is used here?

Reply:
@maddyscientist for insight